Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -184,6 +185,41 @@ private class DisconnectSetting: WithAccountSetting {
     }
 }
 
+private class SyncNowSetting: WithAccountSetting {
+    var syncNowTitle = NSAttributedString(string: NSLocalizedString("Sync Now", comment: "Sync Firefox Account"), attributes: [NSForegroundColorAttributeName: UIColor.grayColor(), NSFontAttributeName: UIFont.systemFontOfSize(UIConstants.DefaultStandardFontSize, weight: UIFontWeightMedium)])
+
+    override var accessoryType: UITableViewCellAccessoryType { return .None }
+
+    override var style: UITableViewCellStyle { return .Value1 }
+
+    override var title: NSAttributedString? {
+        return syncNowTitle
+    }
+
+    override func onConfigureCell(cell: UITableViewCell) {
+        cell.textLabel?.attributedText = title
+        cell.accessoryType = accessoryType
+        cell.accessoryView = nil
+        self.cell = cell
+    }
+
+    override func onClick(navigationController: UINavigationController?) {
+        self.cell?.userInteractionEnabled = false
+        self.cell!.textLabel?.attributedText = NSAttributedString(string: NSLocalizedString("Syncing...", comment: "Syncing Firefox Account"), attributes: [NSForegroundColorAttributeName: UIConstants.ControlTintColor])
+        profile.syncManager.syncEverything().uponQueue(dispatch_get_main_queue()) { result in
+            if result.isSuccess {
+                println("Sync Success")
+            } else {
+                println("Sync Fail")
```

Comment:
Also, use `log.debug`, cargo-culting the definition of `log` from elsewhere if necessary. We shouldn't use `println` anywhere in our codebase.